### DISCUSSION TITLE
Find out more about logs

### DISCUSSION THREAD

**Discussion by FabelxDev** [community/community#117003] [Other Feature Feedback, Questions, & Ideas]
A user got removed from my repository and I want to find out more about it.

Where can i see that information?

**Comment by AlexanderFRT** [comment]
You can try to find the answer of why a user was removed by checking the steps below

1. **Pull Request and Issue Comments:**
   - Check the comments section of pull requests and issues within your repository. Users might leave comments indicating their actions, including the removal of another user.

2. **Collaborator List Changes:**
   - If the removed user was a collaborator on the repository, check the collaborator list under repository settings. Look for recent changes to the list, as this might indicate when the user was removed.

3. **Contributor Graph:**
   - GitHub provides a contributor graph showing the activity of contributors over time. Sudden changes in the graph could indicate when a user's contributions stopped abruptly.

4. **Version Control History:**
   - Use Git commands to examine the version control history of your repository. While this won't directly show user removals, you might find commit messages or changes related to access control that provide insights.

5. **Reach Out to GitHub Support:**
   - If you're unable to find the information you need through the above methods, consider reaching out to GitHub support for assistance. They may be able to provide additional insights or access to logs not available through the web interface.
